src/components/PopularityFilterCollapse.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Collapse, Checkbox } from "@baltimorecounty/dotgov-components";
+
+const PopularityFilterCollapse = props => {
+  const { id, header, onChange, isChecked, isExpanded } = props;
+  return (
+    <div className="dg_accordion">
+      <Collapse id={id} header={header}>
+        <Checkbox
```

Comment:
i believe id and name are required for checkbox.

Reply:
one way to see this is to try and click the label, the expected behavior is that the checkbox will be checked, but in this case, nothing happens.

Reply:
since you are just passing props here, you probably can just use Collapse and Checkbox in the ServiceList

Reply:
done